### DISCUSSION TITLE
Combine icons when taskbar is full

### DISCUSSION THREAD

**Discussion by SS198** [valinet/ExplorerPatcher#2416] [General]
Combine icons when taskbar is full does not work under following settings:
Taskbar style: Windows 10
taskbar location: left

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Reply by SS198** [reply]
Windows 11, Version 22H2, Build 22621.2428
ExplorerPatcher: 22621.2428.59.1

Settings:
Style: Windows 10
Primary taskbar location: left
Start button style: windows 11
Combine taskbar icons: combine when full

![image](https://github.com/valinet/ExplorerPatcher/assets/150333853/acc4776b-99c4-4828-9d95-784775b44be7)

**Reply by pyrates999** [reply]
Try changing the start button style to windows 10.

**Reply by SS198** [reply]
I have tried it but no change in behavior

**Reply by pyrates999** [reply]
Did you click on restart file explorer?

**Reply by SS198** [reply]
yes, I did. Also restarted my PC.
I also changed the setting to Always Combine:
![image](https://github.com/valinet/ExplorerPatcher/assets/150333853/87aa2cfd-e485-44d0-9825-939afbb4dfb0)

still not combining:
![image](https://github.com/valinet/ExplorerPatcher/assets/150333853/a4871981-e910-4be3-b012-977897c73af2)

**Reply by pyrates999** [reply]
Windows 11 now has a feature to never combine actually.  Here's an article on it and how to enable it:

https://www.bleepingcomputer.com/news/microsoft/windows-11s-new-never-combine-icons-feature-is-almost-usable/

**Reply by SS198** [reply]
but I want to combine the icons when taskbar is full, so the above article does not help.

**Reply by pyrates999** [reply]
Let's see if one of the developers can respond then.

**Reply by Amrsatrio** [reply]
So Always combine does not work or what? Combine icons when taskbar is full works if the taskbar is choked. Try spamming Explorer windows and see for yourself.

**Reply by SS198** [reply]
for me both options are not working.

![image](https://github.com/valinet/ExplorerPatcher/assets/150333853/3d761149-8af4-46b1-b93a-5182b6dfc376)

![image](https://github.com/valinet/ExplorerPatcher/assets/150333853/5045d570-8224-4b5b-b4a4-fdf1cddafe54)

**Comment by PK232** [comment]
ExplorerPatcher 2261.2428.59.1
Windows 23H2 Build 22631.2428

I agree with SS198.    If when using PatcherExplorer, I select “Windows 10”,  I am able to have the QuickLaunch Toolbar.   On the other hand the option to combine taskbar icons (Settings/Personalization/Taskbar/Taskbar behaviors/Combine taskbar buttons and hide labels/Never or …./When taskbar is full)  does not work.   

If however I select “Windows 11 (default)”  the taskbar setting to combine icons does work, but I lose the Quick Launch Toolbar.   :(

**Comment by pyrates999** [comment]
Considering you can pin apps to the taskbar, I don't see a reason for the quick launch toolbar.

**Comment by PK232** [comment]
To each his own.  I personally like the way the Quick Launch task bar works and it would be nice if you did not have to choose between the Quick Launch or the combined icons.
